docs/_posts/2016-11-16-react-v15.4.0.md

```diff
@@ -41,7 +41,7 @@ To use it:
 
 Note that the numbers are relative so components will render faster in production. Still, this should help you realize when unrelated UI gets updated by mistake, and how deep and how often your UI updates occur.
 
-Currently Chrome is the only browser supporting this feature, but we use the standard [User Timing API](https://developer.mozilla.org/en-US/docs/Web/API/User_Timing_API) so we expect more browsers to add support for it.
+Currently Chrome, Edge, and IE are the only browsers supporting this feature, but we use the standard [User Timing API](https://developer.mozilla.org/en-US/docs/Web/API/User_Timing_API) so we expect more browsers to add support for it.
```

Comment:
Can we specify that this is IE11, since IE9 and IE10 are still supported by React.